docker_files/inferCNV-alpha

```diff
@@ -0,0 +1,38 @@
+# Docker file for inferCNV
+FROM ubuntu:xenial
+MAINTAINER ttickle@broadinstitute.org
+RUN echo "deb http://cran.rstudio.com/bin/linux/ubuntu xenial/" | tee -a /etc/apt/sources.list && \
+gpg --keyserver keyserver.ubuntu.com --recv-key E084DAB9 && \
+gpg -a --export E084DAB9 | apt-key add -
+RUN apt-get update && \
```

Comment:
Does apt-get leave downloaded files around that we might want to eliminate (&& rm -rf /var/lib/apt) to reduce images size?

Reply:
It does,  I can add the following at the end to clean up:

RUN apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*